the_ark/s3_client.py

```diff
@@ -163,17 +154,20 @@ def get_all_filenames_in_folder(self, path_to_folder, sort_by_date=True):
         return key_list
 
     def get_most_recent_file_from_s3_key_list(self, key_list):
-        # --- Sort out the most recent file for the list of keys
+        """
+        Sorts through the list of files in s3 key list object and returns the most recently modified file in the list
+        :param
+            - key_list:    list - The list of files returned from a s3.bucket.list() operation
+        :return
+            - key   The most recently modified file in the key list
```

Comment:
The key return value isn't formatted like the other return values, not sure if this is a big deal or not.